documentation/source/building.rst

```diff
@@ -62,6 +62,17 @@ Make Variables
 
       Passed to both the compile and execute phases of simulators with two rules, or passed to the single compile and run command for simulators which don't have a distinct compilation stage.
 
+.. make:var:: PLUSARGS
+
+      "Plusargs" are options that are starting with a plus (``+``) sign. 
+      They are passed to the simulator and are also available within cocotb as :attr:`cocotb.plusargs`.
+      In the simulator, they can be read by the (System)Verilog system functions 
```

Comment:
You can skip the parentheses, the language is called SystemVerilog since ~ 10 years now.

Reply:
I wanted to express that this is present in both plain old Verilog and SystemVerilog. I have rewritten this a bit now.